gitlab.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 const (
-	libraryVersion = "0.1.1"
+	libraryVersion = "0.1.2"
 	defaultBaseURL = "https://gitlab.com/api/v3/"
```

Comment:
I noticed you first changed this `v4`, but changed it back again. I guess it's fine to change this to `v4` as this branch is for the v4 API.